app/views/browse/build-config.html

```diff
@@ -121,7 +121,7 @@ <h1 class="contains-actions">
 
                         <button
                             class="btn btn-primary btn-lg"
-                            ng-click="startBuild(buildConfig.metadata.name)"
+                            ng-click="startBuild()"
```

Comment:
The controller uses `$scope.buildConfig` when it calls `BuildsService.startBuild`, so this is OK.